python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb

```diff
@@ -114,36 +114,7 @@
         expect { subject }.
           to raise_error(Dependabot::DependencyFileNotResolvable) do |error|
             expect(error.message).to start_with(
-              "CRITICAL:pipenv.patched.notpip._internal.index:"\
-              "Could not find a version that satisfies the requirement "\
-              "pytest==10.4.0"
-            )
-          end
-      end
-    end
-
-    context "when the Python version conflicts with another dependency" do
-      let(:pipfile_fixture_name) { "unresolvable_python_version" }
-      let(:dependency_files) { [pipfile] }
-
-      let(:dependency_name) { "pytest" }
-      let(:dependency_version) { "3.4.0" }
-      let(:dependency_requirements) do
-        [{
-          file: "Pipfile",
-          requirement: "==3.4.0",
-          groups: ["develop"],
-          source: nil
-        }]
-      end
-
-      it "raises a helpful error" do
-        expect { subject }.
-          to raise_error(Dependabot::DependencyFileNotResolvable) do |error|
-            expect(error.message).to start_with(
-              "pipenv.patched.notpip._internal.exceptions."\
-              "UnsupportedPythonVersion: futures requires Python '>=2.6, <3' "\
-              "but the running Python is 3."
+              "ERROR: No matching distribution found for pytest==10.4.0"
```

Comment:
This error message has changed

Reply:
Looks like this test is failing with a different error message https://github.com/dependabot/dependabot-core/pull/3817/checks?check_run_id=2782969485

Reply:
Yeah I'm looking into that now, the test is passing on my machine so I'm not quite sure what's going on yet